js/classes/entities/foreground/surface_snow.js

```diff
@@ -0,0 +1,55 @@
+class SurfaceSnow {
+    constructor() {
+        this.pos = new Vector(0, 0);
+
+        this.animator = new Animator(Precipitation.SPRITESHEET, new Vector(0, 7 * Precipitation.SIZE.x), new Vector(Precipitation.SIZE.x, Precipitation.SIZE.y), 1, 1);
+
+    }
+    static get SPRITESHEET() {
+        return "./sprites/snow_and_rain.png";
```

Comment:
Should this be a constant in the utils file? I saw other similar paths in there.

Reply:
That is a good question. We have some other classes which have their own sprite sheets. I'll look into that.

Reply:
After asking @NathanHinthorne he reminded me that we put sprite sheets into util if it doesn't clearly belong to any specific class.

Reply:
OK, sounds good